plugintest/snat_test.go

```
@@ -0,0 +1,108 @@
+		It("snat SetSnat", func() {
+
+			cmd := plugin.SetSnatCmd{
+				Snat: env.snat1,
+			}
+			plugin.SetSnat(&cmd)
+
+			b := utils.Bash{
+				Command: "iptables-save -t nat",
+				Sudo:    true,
+			}
+			b.Run()
```

golangci-lint

Error return value of `b.Run` is not checked

(errcheck)